lib/plugins/aws/invokeLocal/index.js

```diff
@@ -276,8 +276,8 @@ class AwsInvokeLocal {
       const ruby = spawn(runtime,
         [path.join(__dirname, 'invoke.rb'), handlerPath, handlerName],
         { env: process.env }, { shell: true });
-      ruby.stdout.on('data', (buf) => this.serverless.cli.consoleLog(buf.toString()));
```

Comment:
Initially we piped everything through `cli.consoleLog` so that we have full control of the output and its formatting. Is there a different way how we could still leverage `consoleLog`?

Reply:
It looks as though `cli.consoleLog` delegates straight to `console.log` - which apparently always adds a newline.

See: https://github.com/serverless/serverless/blob/master/lib/utils/log/consoleLog.js#L4 and https://stackoverflow.com/questions/6157497/node-js-printing-to-console-without-a-trailing-newline

Would this be a misuse of `consoleLog` though in a way? i.e. it's not really a log that's happening here - just echoing/piping output. I could see how with a log, you might want to print a timestamp, log level etc (which would be a reason for a logging function like `consoleLog`, but that doesn't apply here perhaps.

But happy to take guidance since I don't know this framework very well :-)